runtime.txt

```diff
@@ -1 +1 @@
-python-3.8.2
\ No newline at end of file
+python-3.8.2
```

Comment:
I don't see any difference. What changed here? 🤔

Reply:
It added a newline.  I'm not sure why GitHub makes it difficult to see whitespace changes sometimes.